client/admin_test.go

```diff
@@ -0,0 +1,123 @@
+// Copyright 2021 Zenauth Ltd.
+// SPDX-License-Identifier: Apache-2.0
+
+package client
+
+import (
+	"context"
+	"errors"
+	"io"
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/require"
+	"google.golang.org/grpc"
+
+	auditv1 "github.com/cerbos/cerbos/api/genpb/cerbos/audit/v1"
+	responsev1 "github.com/cerbos/cerbos/api/genpb/cerbos/response/v1"
+	svcv1 "github.com/cerbos/cerbos/api/genpb/cerbos/svc/v1"
+)
+
+func TestCollectLogs(t *testing.T) {
+	type ticker struct {
+		tick <-chan time.Time
+		tock <-chan time.Time
+	}
+
+	newTicker := func() ticker {
```

Comment:
What is the purpose of the tickers here? Can't we achieve the same effect quicker by creating a pre-filled channel (or slice) of `ListAuditLogEntriesResponse` as the source for the receiver functions?

Reply:
Yeah we could, I was just trying to keep the order but we can simplify it 👍